Extension.php

```diff
@@ -11,10 +12,40 @@ public function initialize()
     {
         $this->addCss('assets/extension.css');
         $this->addJavascript('assets/start.js', true);
```

Comment:
Stock standard for a `before()`

```
/**
 * Before middleware function.
 *  
 * @param Request     $request
 * @param Application $app
 */
public function before(Request $request, Application $app)
```

It **might** be useful to do something in there, as it isn't well documented by Silex.  But your call for this PR.